app/controllers/reviews_controller.rb

```diff
@@ -18,8 +20,11 @@ def update
   end
 
   def create
-    @review = Review.create(shelter_id: shelter_id[:id])
-    @review.update(review_params)
+    @shelter = Shelter.find(params[:id])
+    @review = @shelter.reviews.create(review_params)
```

Comment:
I think this is cleaner

Reply:
![](https://media.giphy.com/media/30pjt9I0SDs7R4ghlp/giphy.gif)